content/community/code_of_conduct.rst

```diff
@@ -0,0 +1,62 @@
+:date: 2019-09-05 15:00
+:modified: 2019-09-05 16:12
```

Comment:
Note that this is more like a _page_ from *pelican*'s perspective than an _article_. Does that change any of the meta data that we're using here?

Reply:
@bdbaddog I noticed that many of our older pages are in Markdown (`.md`) while our more current things are in ReStructured Text (`.rst`). I used *rst* for this page.

Reply:
@glenjarvis - Either can be used with pelican. (rst or md)